File: music/src/main/java/com/xogrp/john/music/MusicApplication.java
package com.xogrp.john.music;

import android.app.Application;

import com.xogrp.john.music.utils.LogUtil;

/**
 * Created by john on 03/03/2017.
 */

public class MusicApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        LogUtil.e("MusicApplication onCreate");
    }
}

File: musicinkotlin/src/main/java/com/example/musicinkotlin/MusicApplication.kt
package com.example.musicinkotlin

import android.app.Application
import com.tencent.bugly.crashreport.CrashReport

/**
 * Created by john on 12/02/2018.
 */

public class MusicApplication : Application(){

    override fun onCreate() {
        super.onCreate()
        CrashReport.initCrashReport(applicationContext, "7d88acbfa6", true)
    }

}